Mathlib/Topology/Algebra/UniformGroup.lean

```diff
@@ -374,15 +374,6 @@ theorem Filter.HasBasis.uniformity_of_nhds_one_inv_mul_swapped {ι} {p : ι →
 #align filter.has_basis.uniformity_of_nhds_one_inv_mul_swapped Filter.HasBasis.uniformity_of_nhds_one_inv_mul_swapped
 #align filter.has_basis.uniformity_of_nhds_zero_neg_add_swapped Filter.HasBasis.uniformity_of_nhds_zero_neg_add_swapped
 
-@[to_additive]
-theorem group_separationRel (x y : α) : (x, y) ∈ separationRel α ↔ x / y ∈ closure ({1} : Set α) :=
```

Comment:
Aren’t we loosing some mathematical content here?

Reply:
I think the new `group_inseparable_iff` takes care of that.

Reply:
Ok great. I looked for it but couldn’t find it.